### ISSUE TITLE
Add Redeemers to functionality to build unbalanced transaction

### ISSUE THREAD

**Issue by vvtran** [Plutonomicon/cardano-browser-tx#167]
- We need the Haskell server to convert `PlutusData` wrapped `Redeemer`s to indexed redeemers. Once this has been implemented, we need to tweak the constraint processor to incorporate this.

**Comment by ngua** [comment]
Blocked on #156 for now

**Comment by ngua** [comment]
Just writing down a few notes on how we can do this so I don't forget:

- Convert the redeemers to the `Types.Transaction` style and set the execution units in each one to zero (we have every other piece of information we need)
- Attach the redeemers, datums, etc... to the transaction. At this stage the missing ex units in the redeemers is OK (according to Ogmios docs)
- Convert the transaction (it can still be unbalanced at this stage) to hex-encoded string and call Ogmios protocol for evaluating the tx without submitting it

**Comment by vvtran** [comment]
Created a separate issue for Ogmios https://github.com/Plutonomicon/cardano-browser-tx/issues/174 as we are hardcoding the ex units for now.

**Comment by vvtran** [comment]
Closing for the new issue of using Ogmios
